src/CabanaPD_Types.hpp

```diff
@@ -72,7 +72,7 @@ struct LinearLPS
 {
 };
 
-struct NoOutput
+struct BaseOutput
 {
 };
 struct OutputEnergy
```

Comment:
Probably best to keep the naming consistent, i.e. `EnergyOutput` <--> `BaseOutput`